src/app.js

```diff
@@ -61,6 +59,7 @@ configUtil.setConfig('SEARCH_SERVICE_AUTHORIZATION_TOKEN', 'Bearer ' + searchSer
 configUtil.setConfig('DIAL_REPO_AUTHORIZATION_TOKEN', 'Bearer ' + dialRepoApiKey)
 configUtil.setConfig('PLUGIN_REPO_AUTHORIZATION_TOKEN', 'Bearer ' + pluginRepoApiKey)
 configUtil.setConfig('DATA_SERVICE_AUTHORIZATION_TOKEN', 'Bearer ' + dataServiceApiKey)
+configUtil.setConfig('SERVICE_LOGGER_ENABLED', loggerEnabled)
```

Comment:
Where is used this config to enable and disable logs

Reply:
in sunbird logger utils we are getting and using this config